File: Examples/Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/StatDataResponse.java
package com.aspose.words.model;

import com.aspose.words.model.Link;
import com.aspose.words.model.DocumentStatData;
public class StatDataResponse {
  private DocumentStatData StatData = null;
  private Link DocumentLink = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getStatData
	 * Gets DocumentStatData
	 * @return StatData
	 */
  public DocumentStatData getStatData() {
    return StatData;
  }

	/**
	 * setStatData
	 * Sets DocumentStatData
	 * @param StatData DocumentStatData
	 */
  public void setStatData(DocumentStatData StatData) {
    this.StatData = StatData;
  }

  /**
	 * getDocumentLink
	 * Gets Link
	 * @return DocumentLink
	 */
  public Link getDocumentLink() {
    return DocumentLink;
  }

	/**
	 * setDocumentLink
	 * Sets Link
	 * @param DocumentLink Link
	 */
  public void setDocumentLink(Link DocumentLink) {
    this.DocumentLink = DocumentLink;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class StatDataResponse {\n");
    sb.append("  StatData: ").append(StatData).append("\n");
    sb.append("  DocumentLink: ").append(DocumentLink).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/Aspose.Words.Cloud.Sdk/Model/StatDataResponse.cs
namespace Aspose.Words.Cloud.Sdk.Model 
{
  using System;  
  using System.Collections;
  using System.Collections.Generic;
  using System.Runtime.Serialization;
  using System.Text;
  using Newtonsoft.Json;
  using Newtonsoft.Json.Converters;

  /// <summary>
  /// Response for the request of the document&#39;s statistical data
  /// </summary>  
  public class StatDataResponse : SaaSposeResponse 
  {                       
        /// <summary>
        /// Statistical data of the document
        /// </summary>  
        public DocumentStatData StatData { get; set; }

        /// <summary>
        /// Link to the document
        /// </summary>  
        public FileLink DocumentLink { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class StatDataResponse {\n");
          sb.Append("  StatData: ").Append(this.StatData).Append("\n");
          sb.Append("  DocumentLink: ").Append(this.DocumentLink).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
